IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -46,6 +46,13 @@
 
 #import "MSIDAuthenticationScheme.h"
 
+typedef NS_ENUM(NSInteger, MSIDRefreshTokenTypes)
+{
+    MSIDAppRefreshTokenType = 0,
+    MSIDFamilyRefreshTokenType,
+    MSIDInvalidRefreshTokenType
```

Comment:
what is the invalid refresh token type? just curious.

Reply:
Removed